frontend/src/components/scan/ScanForm.jsx

```diff
@@ -250,6 +250,7 @@ export default function ScanForm() {
     () =>
       playbooksGrouped[classification]
         .map((v) => ({
+          isDisabled: !v.verification.configured || v.disabled,
```

Comment:
this is copy/pasted from analyzers/connectors. Playbooks do not have `!v.verification.configured` value. It is enough to set the second clause